odtp/cli/s3.py

```diff
@@ -31,7 +31,7 @@ def download(
 def check():  
     try:
         s3 = s3Manager()
-        bucket = s3.test_connection()
+        s3.test_connection()
```

Comment:
@caviri I added a commit to correct this call, since you are not returning the bucket any more. Now it should be fine.

Reply:
True, I missed that. Thanks!